src/Essentials/src/Geolocation/Geolocation.android.cs

```diff
@@ -30,6 +30,19 @@ partial class GeolocationImplementation : IGeolocation
 		/// </summary>
 		public bool IsListeningForeground { get => continuousListener is not null; }
 
+		public bool IsLocationEnabled
```

Comment:
Barometer and Accelerometer have IsSupported. Would that be a better name for this to be more consistent?